processing-common/src/main/kotlin/com/livefront/sealedenum/internal/common/spec/SealedEnumFileSpec.kt

```diff
@@ -57,7 +57,7 @@ public data class SealedEnumFileSpec(
         }
 
         sealedEnumOptions.forEach { (treeTraversalOrder, sealedEnumOption) ->
-            val sealedObjects = getSealedObjectsFromNode(treeTraversalOrder, sealedClassNode)
+            val sealedObjects = getSealedObjectsFromNode(treeTraversalOrder, sealedClassNode).distinct()
```

Comment:
Can we move this into the implementation of `getSealedObjectsFromNode`? And then update the documentation for that method to note that only unique objects are returned in the list. I think it makes sense to keep all of the "accumulate the list via traversing" logic in the same place.

Reply:
Done!